modules/minileven/minileven.php

```diff
@@ -34,8 +34,6 @@ function jetpack_check_mobile() {
 		return false;
 	if ( jetpack_mobile_exclude() )
 		return false;
-	if ( 1 == Jetpack_Options::get_option_and_ensure_autoload( 'wp_mobile_disable', '0' ) )
```

Comment:
I wouldn't. In sites < 4.2 this option wouldn't even be set. In sites >= 4.2, it's neglectable. The thing is that we would have to add some checking to see if the option was created or not and delete it, and don't want to add more processing time to clean up something so small.